core/controllers/admin_test.py

```diff
@@ -106,6 +108,53 @@ def test_change_about_page_config_property(self):
         self.assertIn(new_config_value, response.body)
 
 
+    def test_add_notification_email_for_failed_tasks(self):
```

Comment:
This test was not really needed. We already check config properties in above tests. You can keep it but knowing this can save your time from next time.